### DISCUSSION TITLE
Clips & Screenshots - Storage (Unraid)

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#1245] [General]
Hi, I am currently running Frigate on a container within Unraid. I have 5 cameras with clips & screenshots turned on. 

During the last 16 days I have had 3060 screenshots & 2421 video clips saved in my clips folder - total 5481 files at 90.9 gigs. 

I don't mind at all taking up the space & I have found the person detection so much better than my previous BlueIris / DeepStack setup, while using much less resources. 

To my questions: 

1) Is there a way to split the screenshots and clips into different folders?  
2) Is there a way the files can either be put into folders, say per day/camera, or split once they get too many? 

It is just 5000+ files in one folder is a bit much.

Thanks

**Comment by blakeblackshear** [comment]
This is intended to be internal storage alongside the database file. It's not really designed to be browsed manually. You can move those files around, but then many features in the web UI will be broken. 

Can you explain why you need to organize them differently?

**Reply by DrSpaldo** [reply]
In comparison to the recordings folder system there are many many folders, ie . \frigate\recordings\2021-06\18\12\backdoor\00.309.mp4 I was thinking there may be a folder convention. It is a stark contrast between the two.

I appreciate that may not be reason enough for you though. 

My main reason would be when I am trying to look for a screenshot, as they do not appear in the HA media browser, only clips. It is quite a bit to scroll through in one folder.

**Reply by blakeblackshear** [reply]
You can download the snapshot from the UI as well where you have filters for camera/zone/object. 

The recordings are organized that way in order to support the way I integrated them into Home Assistant.

**Reply by DrSpaldo** [reply]
I will try using the UI when I am not home, I just haven't found way to integrate it into HA itself, I have been using the separate link as I have HA on one machine and Frigate on my Unraid server. Any suggestions?

**Reply by blakeblackshear** [reply]
Are you using HassOS? Or just the container?

**Reply by DrSpaldo** [reply]
HassOS inside proxmox container. I want to move it over to my Unraid server but I’m scared I’ll break it!

**Reply by mitchross** [reply]
You can access the frigate UI from unraid

Goto the 192.x.x.x:5000 address  ( or whatever it is for you ) 

![image](https://user-images.githubusercontent.com/6330506/122575945-35cb1f80-d01f-11eb-90c4-f6ebfb35c428.png)


In depth if needed - > https://mitchross09.medium.com/how-i-self-host-my-own-sites-and-applications-with-unraid-docker-authelia-and-cloudflare-1f0d8e6f8912

**Reply by DrSpaldo** [reply]
Yep, it’s on an internal IP so I cant just access it within the HA UI. I’ll check out your link and see if I can set up my server with that method

**Reply by blakeblackshear** [reply]
@DrSpaldo There are a few ways to integrate. Are you using [the integration](https://github.com/blakeblackshear/frigate-hass-integration)? Integrating the UI as a panel is only supported for Addons at the moment, but I will be adding a proxy addon for those running frigate on a separate machine.

**Reply by DrSpaldo** [reply]
Hi @blakeblackshear yes I am running the integration, however, frigate is on another machine and not running as an HA addon.

**Reply by blakeblackshear** [reply]
@DrSpaldo I just added a new Frigate Proxy addon: https://github.com/blakeblackshear/frigate-hass-addons

**Reply by DrSpaldo** [reply]
> @DrSpaldo I just added a new Frigate Proxy addon: https://github.com/blakeblackshear/frigate-hass-addons

Great news @blakeblackshear . I've installed it and appears to be working perfect. Thank you very much :)

**Reply by DrSpaldo** [reply]
Hi @blakeblackshear , I was wondering if the sidebar icon for Frigate Proxy was restricted to admins or any reason you can think of why it isn't showing up as an option for my tablet user?

**Reply by blakeblackshear** [reply]
No idea.

**Reply by DrSpaldo** [reply]
So I made the tablet user an admin and then I could see the sidebar icon for the Frigate Proxy. I am not sure what can be changed to allow all users to see it, as I would prefer the tablet not to be an admin?
